### PR TITLE
Enabled Serbian typography rules

### PR DESCRIPTION
**Pull request by strn**
Serbian as language exists in CRengine.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6596)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

File: frontend/apps/reader/modules/readertypography.lua [hunk 1]
```diff
@@ -41,6 +41,7 @@ local HYPH_DICT_NAME_TO_LANG_NAME_TAG = {
     ["Russian_EnGB.pattern"] = { _("Russian + English (UK)"), "ru-GB" },
     ["Russian_EnUS.pattern"] = { _("Russian + English (US)"), "ru-US" },
     ["Russian.pattern"]      = { _("Russian"),      "ru" },
+    ["Serbian.pattern"]      = { _("Serbian"),      "sr" },
```
Comment (poire-z) [inline frontend/apps/reader/modules/readertypography.lua:44]: We don't have any Serbian.pattern in https://github.com/koreader/crengine/tree/master/cr3gui/data/hyph/ 
The fact you added it here is may be a non-issue (it's for migration) - but I think then you could just let it out.

File: frontend/apps/reader/modules/readertypography.lua [hunk 2]
```diff
@@ -86,6 +87,7 @@ local LANGUAGES = {
     { "ru-GB",                 {},   "Hb  ",   _("Russian + English (UK)") },
     { "ru-US",                 {},   "Hb  ",   _("Russian + English (US)") },
     { "ru",               {"rus"},   "Hb  ",   _("Russian") },
+    { "sr",               {"srp"},   "H   ",   _("Serbian") },
```
Comment (poire-z) [inline frontend/apps/reader/modules/readertypography.lua:90]: And if you're no bringing a Serbian.pattern, just remove the H from the features string (it's for display in the menu, describing what features are available). Let it as for "ja" and "ko".

File: frontend/apps/reader/modules/readertypography.lua (hunk 1 above)
Comment (Frenzie) [reply frontend/apps/reader/modules/readertypography.lua:44]: Thanks, now I'm confused by this PR.

File: frontend/apps/reader/modules/readertypography.lua (hunk 2 above)
Comment (strn) [reply frontend/apps/reader/modules/readertypography.lua:90]: Sorry about the confusion. Can you please suggest what else I need to add in order to proper add typographic rules for Serbian language?

Comment (poire-z) [reply frontend/apps/reader/modules/readertypography.lua:90]: As you added _`duplicate_real_hyphen_on_next_line` for Serbian , you can replace your `H` here with `HB`.

**Comment by poire-z** [comment]
I'm not allowed to push commits to your branch. Can you enable it (somewhere on github)?
I'd like to get this merged, so it does not block koreader-base.
If no news in the afternoon, I'll merge it and do another PR to bump base and update "H" to "HB".

**Comment by strn** [comment]
Sorry for late reply - cannot access email or Github from work. What shall I do? Can you point it to me?

**Comment by poire-z** [comment]
No need to anymore, I did the bump earlier in #6618.
So, it will all be in tomorrow's nightly.

**Pull request merged**